### PR TITLE
UPD

### PR DESCRIPTION
**Pull request by Roman-Braneshty**
Check my beautifully func

### CODE REVIEW

File: utils/units.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Person:
```
Comment (vlad-bb) [inline utils/units.py:1]: good

File: utils/units.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    def __init__(self, name):
```
Comment (vlad-bb) [inline utils/units.py:2]: norm

File: utils/units.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+        self.name = name
```
Comment (vlad-bb) [inline utils/units.py:3]: perfect

File: utils/units.py (hunk 1 above)
Comment (Dimasta22) [inline utils/units.py:1]: Крутой класс

**Review by vlad-bb — APPROVED** [review]
Good job

**Pull request merged**